SmartDeviceLink/SDLTCPTransport.m

```diff
@@ -195,6 +166,13 @@ - (void)sdl_cancelIOThread {
     [self performSelector:@selector(sdl_doNothing) onThread:self.ioThread withObject:nil waitUntilDone:NO];
 }
 
+/// Cancels the connection timer for establishing a TCP socket with the accessory.
+- (void)sdl_cancelConnectionTimer {
+    if (self.connectionTimer == nil) { return; }
```

Comment:
I feel like this `NSTimer` should be an `SDLTimer`, but it could be a separate issue / PR

Reply:
Created issue #1611